tests/e2e_test/utils_test.go

```diff
@@ -50,6 +50,8 @@ const (
 	interval              = 1 * time.Second
 	remoteNamespace       = "kyma-system"
 	controlPlaneNamespace = "kcp-system"
+	moduleName            = "template-operator"
```

Comment:
if you create all test module with `NewTemplateOperator`, we don't need to define this `moduleName` specifically.